pyet/version.py

```diff
@@ -1,3 +1,3 @@
 # This is the only location where the version will be written and changed.
 # Based on https://packaging.python.org/single_source_version/
-__version__ = "1.1.0"
+__version__ = "1.1.1"
```

Comment:
This should be 1.2.0b, cause you merge it into Dev ('b') and it is definitely more than a minor update.